frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -70,8 +70,14 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
       <DashboardCardBody>
         <DetailsBody>
           <DetailItem
-            key="name"
-            title="Name"
+            key="service_name"
+            title="Service Name"
+            value="OpenShift Container Storage"
```

Comment:
this is a downstream name isnt it ? how are we going to handle upstream ?

Reply:
@cloudbehl ?

Reply:
@rawagner It is but they are using the same name for the ocs operator also in the upstream.

https://github.com/openshift/ocs-operator#the-ocs-meta-operator

Reply:
@andybraren @yuvalgalanti @lizsurette ^^

Reply:
@rawagner @cloudbehl AFAIK, the upstream name for the ocs operator is the same as the downstream name at the moment.

@jarrpa obnoxxx  Thoughts?

Reply:
There's a bit of nuance here, I'll try to clear it up.

In my opinion, when people talk generally about upstream vs downstream they are referring a relationship of Kubernetes vs OpenShift. The ocs-operator will never run on straight Kubernetes, as such there will never be an upstream version in that regard. The "upstream" version of OCS is Rook-Ceph and noobaa-operator deployed together.

There is, however, the notion of OKD as the OpenShift upstream. In this case, it would be feasible to have an "upstream" of OCS that pulls official images from quay (or wherever), and in that scenario the ocs-operator would keep the same name.

Reply:
thanks for clarification!

Reply:
:100:% what  @jarrpa said. Summary:
- No ocs-operator in the kubernetes upstream
- ocs-operator under the name of "ocs-operator" in the OKD upstream.